charts/cp-control-center/values.yaml

```diff
@@ -78,8 +78,8 @@ prometheus:
   ## ref: https://github.com/prometheus/jmx_exporter
```

Comment:
Global remark: this PR is modifying a lot of charts, but I'm really not sure we are using all of them.
So, you should probably simplify and only modify the one that are useful :)

Reply:
Ok done 👍